docs/apache-airflow/index.rst

```diff
@@ -15,65 +15,120 @@
 
+Airflow evaluates this script and executes the tasks at the set interval and in the defined order. The status
+of the "demo" DAG is visible in the web interface:
 
-Beyond the Horizon
-------------------
+.. image:: /img/hello_world_graph_view.png
+  :alt: Demo DAG in the Graph View, showing the status of one DAG run
 
-Airflow **is not** a data streaming solution. Tasks do not move data from
-one to the other (though tasks can exchange metadata!). Airflow is not
-in the `Spark Streaming <http://spark.apache.org/streaming/>`_
-or `Storm <https://storm.apache.org/>`_ space, it is more comparable to
-`Oozie <http://oozie.apache.org/>`_ or
-`Azkaban <https://azkaban.github.io/>`_.
+This example demonstrates a simple Bash and Python script, but these tasks can run any arbitrary code. Think
+of running a Spark job, moving data between two buckets, or sending an email. The same structure can also be
+seen running over time:
+
+.. image:: /img/hello_world_grid_view.png
+  :alt: Demo DAG in the Grid View, showing the status of all DAG runs
+
+Each column represents one DAG run. These are two of the most used views in Airflow, but there are several
+other views which allow you to deep dive into the state of your workflows.
+
+Why Airflow?
+=========================================
+Airflow is a batch workflow orchestration platform. The Airflow framework contains operators to connect with
+many technologies and is easily extensible to connect with a new technology. If your workflows have a clear
+start and end, and run at regular intervals, they can be programmed as an Airflow DAG.
+
+If you prefer coding over clicking, Airflow is the tool for you. Workflows are defined as Python code which
+means:
+
+- Workflows can be stored in version control so that you can roll back to previous versions
+- Workflows can be developed by multiple people simultaneously
+- Tests can be written to validate functionality
+- Components are extensible and you can build on a wide collection of existing components
+
+Rich scheduling and execution semantics enable you to easily define complex pipelines, running at regular
+intervals. Backfilling allows you to (re-)run pipelines on historical data after making changes to your logic.
+And the ability to rerun partial pipelines after resolving an error helps maximize efficiency.
+
+Airflow's user interface provides both in-depth views of pipelines and individual tasks, and an overview of
+pipelines over time. From the interface, you can inspect logs and manage tasks, for example retrying a task in
+case of failure.
+
+The open-source nature of Airflow ensures you work on components developed, tested, and used by many other
+companies around the world. In the active `community <https://airflow.apache.org/community>`_ you can find
+plenty of helpful resources in the form of blogs posts, articles, conferences, books, and more. You can
+connect with other peers via several channels such as Slack and a mailing list.
+
```

Comment:
```suggestion
You cam also find out that you are among many companies that use Airflow by exploring `IN_THE_WILD <https://github.com/apache/airflow/blob/main/INTHEWILD.md>` list of companies (and add yours by PR).
```

Reply:
I'd like to keep this page as concise as possible, IMO the overview should really be a one-pager that only tells the most important information about Airflow. What do you think of linking the word "companies" to INTHEWILD.md?

Reply:
Yeah :). Good point with being concise :)